config_src/drivers/solo_driver/MOM_surface_forcing.F90

```diff
@@ -735,13 +742,15 @@ subroutine wind_forcing_from_file(sfc_state, forces, day, G, US, CS)
       if (.not.read_Ustar) then
         if (CS%read_gust_2d) then
           do j=js,je ; do i=is,ie
-            forces%tau_mag(i,j) = CS%gust(i,j) + sqrt(temp_x(i,j)**2 + temp_y(i,j)**2)
-            forces%ustar(i,j) = sqrt(forces%tau_mag(i,j) * US%L_to_Z / CS%Rho0)
+            tau_mag = CS%gust(i,j) + sqrt(temp_x(i,j)**2 + temp_y(i,j)**2)
```

Comment:
Not a frequent operation, so I won't hold it up, but the `if (associated())` checks are probably slowing down the loop, and it could have been split as in the (deprecated) `wind_forcing_gyres` loop L534-542